bundles/org.openhab.binding.kodi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -560,4 +562,16 @@
 		<description>Profile of Kodi</description>
 		<state pattern="%s"/>
 	</channel-type>
+	<channel-type id="screensaver">
+		<item-type>Switch</item-type>
+		<label>Screensaver</label>
+		<description>Status of the screensaver</description>
```

Comment:
These texts (and inputrequested) must be added to resources/OH-INF/i18n/kodi.properties as well:
```
channel-type.kodi.screensaver.label = Screensaver
channel-type.kodi.screensaver.description= Status of the screensaver
channel-type.kodi.inputrequested.label = InputRequested
channel-type.kodi.inputrequested.description= Status that shows if Kodi is asking for user input.
```

Reply:
It's already there, isn't it?

Reply:
Right. I somehow missed the file, sorry.